.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         env:
           POLYGON_URL: ${{ secrets.POLYGON_URL }}
 
-      - name: Deploy contract
-        run: yarn hardhat --network hardhat run ./scripts/deployOffsetHelper.ts
-        env:
-          POLYGON_URL: ${{ secrets.POLYGON_URL }}
+      # - name: Deploy contract
+      #   run: yarn hardhat --network hardhat deployOffsetHelper
```

Comment:
We could have a `--verify` flag provided in the `deployOffsetHelper` task to run the verification conditionally and set it here to `false`.

Reply:
Makes sense. I'll amend the above commit